### ISSUE TITLE
[GeoScribble] Long strings don't fit buttons

### ISSUE THREAD

**Issue by matheusgomesms** [Zverik/every_door#731]
Wonderful tool, Ilya!!

As a translator of this app, I noticed this: I know in Weblate we have this " Keep it short." description for the Scribble mode buttons, but even though, how short? Didn't know until today, where AppStore update showed up, and saw that many languages are having trouble with space:

![IMG_04A8C2CD01DE-1](https://github.com/Zverik/every_door/assets/86851490/fb180147-92d8-458e-ae9f-31ff95dea0c6)

Some languages where I saw this happening (not full list): Czech, French, German, Portuguese.

What could be the solution? Have a char limit in Weblate, so translators know they have to respect that, or buttons can be adapted?

**Comment by Zverik** [comment]
Yes, this is a problem, a bit related to #715. I shouldn't have expected to have titles in _all_ languages to fit into.. what, 11 letters? These buttons need word wrapping at least.
